src/integrate/scarches/config.vsh.yaml

```diff
@@ -0,0 +1,140 @@
+functionality:
+  name: scarches
+  namespace: "integrate"
+  version: "dev"
+  description: "Performs reference mapping with scArches"
+  authors:
+    - name: Vladimir Shitov
+      email: vladimir.shitov@helmholtz-muenchen.de
+      roles: [ author ]
+      props: { github: vladimirshitov, orcid: "0000-0002-1960-8812" }
+  argument_groups:
+    - name: Inputs
+      arguments:
+        - name: "--input"
+          alternatives: ["-i"]
+          type: file
+          description: Input h5mu file to use as a query
+          direction: input
+          required: true
+        - name: "--modality"
+          type: string
+          default: "rna"
+          required: false
+        - name: "--reference"
+          choices: ["HLCA"]
+          alternatives: ["-r"]
+          type: string
+          description: Name of the reference dataset
+          required: false
+        - name: "--unlabeled_category"
+          type: string
+          description: "Value used for unlabeled cells"
+          default: "Unknown"
+          required: false
+        - name: "--labels_key"
+          type: string
+          description: "Name of the column containing cell types"
+          default: "leiden"
+          required: false
+        - name: "--predicted_labels_key"
+          type: string
+          description: Name of the column that will contain cell types predicted by scANVI
+          default: "predicted_labels"
+          required: false
+        - name: "--input_layer"
+          type: string
+          required: false
+          description: "Input layer to use. If None, X is used"
+        - name: "--obs_batch"
+          type: string
+          default: "sample_id"
+          required: false
+          description: Column name discriminating between your batches.
+        - name: "--var_input"
+          type: string
+          required: false
+          description: ".var column containing highly variable genes. By default, do not subset genes."
+    - name: Outputs
+      arguments:
+        - name: "--output"
+          alternatives: ["-o"]
+          type: file
+          description: Output h5mu file.
+          direction: output
+          required: true
+        - name: "--obsm_output"
+          type: string
+          default: "X_integrated_scanvi"
+          required: false
+          description: "In which .obsm slot to store the resulting integrated embedding."
+    - name: "Early stopping arguments"
```

Comment:
do we need this for scArches? I don't think so... so not needed for workflows 1 and 2 ref mapping

Reply:
unless needed for fine-tuning?